pkg/api/bzzchunk.go

```diff
@@ -7,17 +7,23 @@ package api
 import (
 	"bytes"
 	"errors"
+	"fmt"
+	"github.com/ethersphere/bee/pkg/tags"
```

Comment:
Somehow this import is among the stdlib's packages. Not so important, but it would be nice to have it a few lines below within the bee imports. Also noticed in some other places in this pr...